src/chat-bubble/index.ts

```diff
@@ -0,0 +1,31 @@
+import { customElement } from "lit/decorators.js";
+import { ChatBubbleBase } from "./chat-bubble-base";
+import baseStyles from "./chat-bubble-base.style";
+import { ChatBubbleIn } from "./chat-bubble-in";
+import inStyles from "./chat-bubble-in.style";
+import { ChatBubbleOut } from "./chat-bubble-out";
+import outStyles from "./chat-bubble-out.style";
+
+export { Parts, Slots } from "./constants";
+
+@customElement("tap-chat-bubble-base")
+export class TapChatBubbleBase extends ChatBubbleBase {
+  public static override readonly styles = [baseStyles];
+}
```

Comment:
Why should we expose the ChatBubbleBase as a public API?

Reply:
Fixed.